week-1/H-numbers/exercise.js

```diff
@@ -1 +1,7 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var NumberOfStudents = 15;
+var NumberofMentors = 8;
+var sum = 15 + 8;
```

Comment:
Can you try to use better name of sum variable (Log a message that displays the total number of students and mentors).
also can you make it more dynamic by refer this variable to the sum of the NumberOfStudents and NumberofMentors variable instead of adding them manually